crates/swc/tests/fixture/issues-10xxx/10442-ignoreList/input/.swcrc

```diff
@@ -0,0 +1,4 @@
+{
+    "sourceMaps": true,
+    "sourceMapIgnoreList": "index\\.js$"
```

Comment:
We discussed making this a regexp but would it be possible to make a Glob? That's probably the more useful thing to do.

Reply:
Yeah, I agree. Currently, SWC does not expose any glob config, but I think it would be worth adding one

Reply:
I'll do with another PR because I'm going to allow `glob` in everywhere we accept regex and that's out of scope of this PR.